index.js

```diff
@@ -24,7 +26,26 @@ module.exports = function () {
 		try {
 			var ret = rework(file.contents.toString(), {source: file.path});
 			plugins.forEach(ret.use.bind(ret));
-			file.contents = new Buffer(ret.toString(options));
+
+			if (file.sourceMap) {
+				options.sourcemap = true;
```

Comment:
You need to run `options` through `Object.assign` on line 12, so you don't modify the user's object.

Reply:
Fixed